src/hotspot/share/gc/shenandoah/shenandoahMmuTracker.cpp

```diff
@@ -94,12 +94,17 @@ void ShenandoahMmuTracker::report() {
   // This is only called by the periodic thread.
   double process_time_s = process_time_seconds();
   double elapsed_process_time_s = process_time_s - _process_reference_time_s;
+  if (elapsed_process_time_s <= 0.01) {
+    // No cpu time for this interval?
+    return;
+  }
+
   _process_reference_time_s = process_time_s;
-  double verify_time_s = gc_thread_time_seconds();
-  double verify_elapsed = verify_time_s - _collector_reference_time_s;
-  _collector_reference_time_s = verify_time_s;
-  double verify_mmu = ((elapsed_process_time_s - verify_elapsed) / elapsed_process_time_s) * 100;
-  _mmu_average.add(verify_mmu);
+  double collector_time_s = gc_thread_time_seconds();
+  double elapsed_collector_time_s = collector_time_s - _collector_reference_time_s;
+  _collector_reference_time_s = collector_time_s;
+  double minimum_mutator_utilization = ((elapsed_process_time_s - elapsed_collector_time_s) / elapsed_process_time_s) * 100;
```

Comment:
Alternative: You could check if result `isnan()` and not use it, although I don't know if divide by zero is costly.

Reply:
I'd just as soon catch it earlier and I'm not 100% certain that other values won't also cause this error. Also, `isnan` won't flag the result of division by zero (`isinf` would though).